Calculator/Calculator/MainPage.xaml

```diff
@@ -7,6 +7,6 @@
     <TabbedPage.Children>
         <local:AbsoluteLayout />
         <local:StackLayout />
-        <local:GridLayout />
+        <local:Grid />
```

Comment:
that's better to name pages something like GridPage, or GridLayoutPage, for example. It's confisung when you give pages same names as standard controls